Modern/ops/src/main/java/org/bonej/ops/CleanShortEdges.java

```diff
@@ -313,8 +314,11 @@ private void euclideanDistance(final Edge e) {
 	}
 
 	private double calibratedLength(final Vector3d centre1) {
-		return Math.sqrt(Math.pow(centre1.x * calibration3d.get(0), 2) + Math.pow(centre1.y * calibration3d.get(1), 2)
```

Comment:
`centre1` is a weird name for a vector whose length we want to calculate? Maybe it should be called `uncalibratedVector3d`?

Reply:
Good catch, but this PR only affected the formatting. In any case, PR #103 deletes this method.